api/model.go

```diff
@@ -800,9 +800,12 @@ func toVolumeResource(v *longhorn.Volume, ves []*longhorn.Engine, vrs []*longhor
 	if v.Status.Robustness == types.VolumeRobustnessFaulted {
 		actions["salvage"] = struct{}{}
 	} else {
+		// attach/detach actions should always be exposed
+		// it's the managers job to determine whether an attach/detach action is valid
+		actions["attach"] = struct{}{}
+		actions["detach"] = struct{}{}
```

Comment:
Need to check what will happen:
1. Detach when detaching
2. Attach when attaching
3. Attach when detaching

Also, this change might confuse the UI. I am a bit concern about this one.